packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionHistorySearchResults.tsx

```diff
@@ -47,9 +56,49 @@ function useTransactionHistoryUpdater() {
 const tabClasses =
   'text-white px-3 mr-2 border-b-2 ui-selected:border-white ui-not-selected:border-transparent ui-not-selected:text-white/80 arb-hover'
 
+const FailedChainPairsTooltip = ({
+  failedChainPairs
+}: {
+  failedChainPairs: ChainPair[]
+}) => {
+  if (failedChainPairs.length === 0) {
+    return null
+  }
+
+  return (
+    <TransactionHistoryStatusBar wrapperClassName="flex space-x-1 bg-error">
```

Comment:
Reusing the current status bar that we have e.g. for pending or claimable transactions